documentation/assemblies/upgrading/assembly-upgrade.adoc

```diff
@@ -22,7 +22,7 @@ Upgrading Strimzi from the previous minor version to version {ProductVersion}.
 Multi-version::
 Upgrading Strimzi from an old version to version {ProductVersion} within a single upgrade (skipping one or more intermediate versions).
 +
-For example, upgrading from Strimzi 0.20.0 directly to Strimzi {ConvertAfterProductVersion}.
+For example, upgrading from Strimzi 0.23.0 directly to Strimzi {ProductVersion}.
```

Comment:
Why did this changed?

Reply:
It was saying 0.20 to 0.22. I thought we might update it to 0.23 to _latest_

Reply:
Ok, that makes sense to me now. Thanks.